pages/intentpreview_test.py

```diff
@@ -248,14 +248,17 @@ class IntentEmailPreviewTemplateTest(testing_config.CustomTestCase):
   def setUp(self):
     super(IntentEmailPreviewTemplateTest, self).setUp()
     self.feature_1 = FeatureEntry(
-        name='feature one', summary='sum', owner_emails=['user1@google.com'],
+        id=1, name='feature one', summary='sum',
+        owner_emails=['user1@google.com'],
         category=1, intent_stage=core_enums.INTENT_IMPLEMENT)
     # Hardcode the key for the template test
     self.feature_1.key = ndb.Key('FeatureEntry', 234)
     self.feature_1.wpt = True
     self.feature_1.wpt_descr = 'We love WPT!'
     self.feature_1.put()
 
+    self.stage_1 = Stage(feature_id=1, stage_type=150, ot_display_name="Test 123")
```

Comment:
Is `Test 123` supposed to show up in the new golden, because I don't see it there.

Reply:
Sorry about this, I shipped this over to you in a hurry without vetting it and it seems there are some issues that need to be handled. I'll re-request a review once this is taken care of.